x-pack/plugin/ilm/src/main/resources/META-INF/services/org.elasticsearch.immutablestate.ImmutableClusterStateHandlerProvider

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
I added this file because we need it to expose the service provider when we load them in the future using PluginsService createExtension.